dataprep/eda/diff/compute/common.py

```diff
@@ -0,0 +1,80 @@
+"""Common types and functionalities for compute(...)."""
```

Comment:
I don't think it's ideal that we copy code. It might be better to put this in `eda/utils.py`.